Godeps/Godeps.json

```diff
@@ -1679,7 +1679,7 @@
 		},
 		{
 			"ImportPath": "github.com/go-sql-driver/mysql",
-			"Comment": "v1.3.0",
+			"Comment": "v1.3",
```

Comment:
@cblecker these comment changes are ok? Or anything broken?

Reply:
for some reason godep-restore does not download all tags here. If I run `git fetch` in each repo, godep-save is happy.